Sources/TuistDependencies/SwiftPackageManager/Utils/PackageInfoMapper.swift

```diff
@@ -657,7 +657,7 @@ extension ResourceFileElements {
         "xcdatamodeld",
         "xcmappingmodel",
         "xcassets",
-        "lproj",
+        "strings",
```

Comment:
Could we keep both?

Reply:
What do you mean? aren't lproj made of strings files?

Reply:
True, I thought we are looking at localized directories and files. Never mind this looks good